contrib/src/uuid.rs

```diff
@@ -0,0 +1,133 @@
+extern crate uuid as uuid_ext;
+
+use std::fmt;
+use std::str::FromStr;
+use std::ops::{Deref, DerefMut};
+use rocket::request::FromParam;
+
+/// The UUID type, which implements `FromParam`. This type allows you to accept
+/// values from the [Uuid](https://github.com/rust-lang-nursery/uuid) crate as
+/// a dynamic parameter in your request handlers.
+///
+/// # Usage
+///
+/// To use, add the `uuid` feature to the `rocket_contrib` dependencies section
+/// of your `Cargo.toml`:
+///
+/// ```toml,ignore
+/// [dependencies.rocket_contrib]
+/// version = "*"
+/// default-features = false
+/// features = ["uuid"]
+/// ```
+///
+/// The UUID type implements the Deref trait allowing you to access the
```

Comment:
Instead of saying this, say that the fact that it implements `FromParam` means you can use it directly as a target of a dynamic parameter. And the example is clear.